### PR TITLE
DOC: AdaptiveHistogramEqualizationImageFilter default radius

### LINKED ISSUE #727: AdaptiveHistogramEqualizationImageFilter doesn't document default radius
Looking at the docs here:
https://itk.org/Doxygen/html/classitk_1_1AdaptiveHistogramEqualizationImageFilter.html

As well as the rendered help from SimpleITK:
```
AdaptiveHistogramEqualization(Image image1, VectorUInt32 radius, float alpha=0.3, float beta=0.3) -> Image
```

The default radius parameter isn't documented.

The filter header does actually set a default value:
https://itk.org/Doxygen/html/itkAdaptiveHistogramEqualizationImageFilter_8h_source.html#163

### PR DESCRIPTION
**Pull request by thewtex**
Closes #727

### CODE REVIEW

**Comment by thewtex** [comment]
@gdevenyi please take a look

**Comment by gdevenyi** [comment]
Looks good :)

File: Modules/Filtering/ImageStatistics/include/itkAdaptiveHistogramEqualizationImageFilter.h [hunk 1]
```diff
@@ -47,7 +47,8 @@ namespace itk
  * (beta=1, with alpha=1).
  *
  * The parameter window controls the size of the region over which
- * local statistics are calculated.
+ * local statistics are calculated. The size of the window is controlled
```
Comment (dzenanz) [inline Modules/Filtering/ImageStatistics/include/itkAdaptiveHistogramEqualizationImageFilter.h]: controlled **by**

Comment (thewtex) [reply Modules/Filtering/ImageStatistics/include/itkAdaptiveHistogramEqualizationImageFilter.h]: Good catch -- fixed!

Comment (gdevenyi) [reply Modules/Filtering/ImageStatistics/include/itkAdaptiveHistogramEqualizationImageFilter.h]: This is why I can't edit for grammar for anyone's paper :-)

**Comment by thewtex** [comment]
Test failures should be unrelated.

**Pull request merged**